src/parser/mod.rs

```diff
@@ -3278,6 +3278,31 @@ impl<'a> Parser<'a> {
         ret
     }
 
+    pub fn parse_actions_list(
+        &mut self,
+    ) -> Result<Vec<(Keyword, Option<Vec<Ident>>)>, ParserError> {
```

Comment:
Getting a clippy warning about `very complex type`, can I just ignore it? or do we prefer aliasing the type?

Reply:
The type is indeed hard to read. I think the warning shouldn't be ignored.

Shouldn't the `Vec<Ident>` be an `ObjectName` ?